### PR TITLE
Formatting improvements & additional information in BTCPi setup docs

### PR DESCRIPTION
**Pull request by ch4ot1c**
Refinements to the Raspberry Pi setup instructions. Ready for review.

This began as indenting the hardware images in the bulleted list, but quickly grew in scope. Focused on clarity and consistency.

Remaining unaddressed topics:
- Estimated setup time
- Pruning configuration
- Additional nodes/coins configuration
- Dynamic DNS configuration
- Default Wi-Fi configuration could be filled in

Layout tested in GH-MD, untested in GitBook.

### CODE REVIEW

File: RaspberryPiDeployment.md [hunk 1]
```diff
@@ -1,110 +1,144 @@
...
+Andreas Antonopoulos tells us "Not your keys, not your Bitcoin". Here's another Bitcoin axiom: "Your hardware. Your node. Your keys. Your Bitcoin."
 
-In order for your Bitcoin experience to be truly self-sovereign and trustless you should consider running nodes on your own hardware and internet connection. BTCPayServer is an excellent way to run both Bitcoin & Lightning nodes.  Not only are you validating transactions you also get the ability to accept base layer Bitcoin and second layer Lightning payments.  
-To that end here are instructions to install and host your very own BTCPay Server on a Raspberry Pi. 
+For your Bitcoin experience to be truly self-sovereign and trustless, you should run nodes on your own hardware and internet connection. BTCPayServer is an excellent way to run Bitcoin & Lightning nodes together. Not only are you validating transactions, you also get the ability to accept Bitcoin (layer 1) and Lightning (layer 2) payments.
 
-The process is basically the following:
+To that end, here are instructions to install and host your very own BTCPayServer on a Raspberry Pi. We call it a BTCPi.
```
Comment (jmaurice) [inline RaspberryPiDeployment.md]: BTCPi? 😂

**Comment by jmaurice** [comment]
Now that Raspberry Pi 4 has been released, these setup instructions should be renamed to the "Raspberry Pi 3" instructions, and a separate document for RPI4 added. For example, RPI3 only has 1GB of ram, and requires swap, but RPI4 has up to 4GB of ram, and doesn't swap or the opt-save-memory option, so there are some significant differences.

**Comment by jmaurice** [comment]
@ch4ot1c Your changes look good, but I think I want to tweak them a tiny bit before merging, so the best workflow is probably for me to create a new task branch, cherry-pick your commit, and then submit that as a new PR after tweaking which contains commits from each of us.

**Comment by ch4ot1c** [comment]
ACK - Go ahead and do so! Thanks

**Comment by wiz** [comment]
Sorry got delayed a few days because of some Bisq work. I just spoke to @britttttk and suggested to merge your changes as-is for now and I'll submit a new PR within a day or so. Thanks @ch4ot1c for your contribution!!

**Pull request merged**